modules/middleware/src/buildAggregations/createFieldAggregation.js

```diff
@@ -25,6 +25,9 @@ const createNumericAggregation = ({ type, field, graphqlField }) => {
 };
 
 const createTermAggregation = ({ field, isNested, graphqlField }) => {
+  const maxAggregations = get(graphqlField, ['buckets', '__arguments', 0], {
+    max: { value: MAX_AGGREGATION_SIZE },
+  }).max.value;
```

Comment:
We can also `get` the value directly using the path array, making it easier to read.

```suggestion
  const maxAggregations = get(
    graphqlField,
    ['buckets', '__arguments', 0, 'max', 'value'],
    MAX_AGGREGATION_SIZE
  );
```

Reply:
done